modules/whos_on_first.py

```diff
@@ -77,20 +76,22 @@ def get_svg(self, led):
             svg += f'<text x="{x}" y="{y}" text-anchor="middle" style="font-family:sans-serif;font-size:16pt;" fill="#000">{text}</text>'
         svg += '</svg>'
         return svg
-    
+
     def randomize(self):
         self.display = random.choice(list(self.DISPLAY_WORDS.keys()))
         self.buttons = random.sample(random.choice(self.BUTTON_GROUPS), 6)
         self.log(f"State randomized. Stage {self.stage}. Display: {self.display}. Buttons: {' '.join(self.buttons)}")
-    
+
+    def button_replace(self,v):
+        return v
+
```

Comment:
I like how you're moving this clearly Third Base-specific code. However, this does not belong in this commit. Besides, make the function name descriptive. How about `canonical_button_name`? Also, add a comment explaining that the function is not a no-op in subclasses.